target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips

```diff
@@ -21,6 +24,13 @@ ramips_set_preinit_iface() {
 			ralink_switchdev=rt305x
 			;;
 		esac
+
+		case "$board" in
```

Comment:
Why is the solution above this not working (--> commit message)?

Reply:
WF2770 has two ethernet switch, one is SoC internal and the other is external MT7530BU. This preinit script disable VLAN for SoC internal switch, but in this case external one should be reset.

Reply:
Please add a comment about that either to commit message or in the code.

Reply:
Added in the code.